core/src/main/java/com/amplifyframework/core/model/ModelSchema.java

```diff
@@ -242,12 +254,27 @@ public String getPluralName() {
     }
 
     /**
-     * Returns true if this model has owner based authorization which changes the parameters for subscriptions
-     * e.g. @auth(rules: [{allow: owner}]) on the model in the GraphQL Schema
-     * @return true if owner authorization is present on this model and false if not or if not explicitly annotated.
+     * Returns list of rules defining which users can access or operate against an object.
+     * e.g. @auth(rules: [{allow: owner}]) on the model in the GraphQL Schema.
+     *
+     * @return List of {@link AuthRule}s for this Model
+     */
+    public List<AuthRule> authRules() {
+        return authRules;
+    }
+
+    /**
+     * Returns true if this model has owner based authorization which changes the parameters for subscriptions.
+     * @return true if owner authorization is present on this model and false if not or if not explicitly annotated
      */
+
     public boolean hasOwnerAuthorization() {
```

Comment:
Is this still necessary? Is it kept because it is `public` and you don't want to change the contract? Should we mark this `@Deprecated`, even if we'll keep it and update its implementation? (I'm not sure?)

Reply:
This is actually a helper method now, and is being used as such.  As I dive in deeper to figure out how to annotate each GraphQLRequest with auth stuff (my next task), I might refactor/move this logic elsewhere, but not totally sure on that yet.